emukit/core/loop/candidate_point_calculators.py

```diff
@@ -37,3 +38,49 @@ def compute_next_points(self, loop_state: LoopState) -> np.ndarray:
         """
         x, _ = self.acquisition_optimizer.optimize(self.acquisition)
         return np.atleast_2d(x)
+
+
+class MultiSourceSequential(CandidatePointCalculator):
+    """
+    This candidate point calculator optimizes the acquisition function separately for each information source,
```

Comment:
what does `separately' mean in that context? sequentially?

Reply:
It means that there are separate optimizations for each information source parameters and we don't jointly optimize the input location and information source parameter.

Reply:
Is this an approximation? I guess no? From the docstring I would not know and would need to ask a developer or go through the code. Basically what I am asking is if `separately' implies that you optimize input location per source sequentially or not, and if yes, if that is an approximation to the solution of the optimization problem. Let's make this more clear.

Reply:
I'll rephrase and move the the `compute_next_points()` method as there is too much detail for class docstring